pages/dkp/konvoy/2.2/choose-infrastructure/pre-provisioned/create-cluster/index.md

```diff
@@ -8,70 +8,83 @@ beta: false
+	configmap/node-feature-discovery-preprovisioned-example created
+	clusterresourceset.addons.cluster.x-k8s.io/nvidia-feature-discovery-preprovisioned-example created
+	configmap/nvidia-feature-discovery-preprovisioned-example created
+	clusterresourceset.addons.cluster.x-k8s.io/metallb-preprovisioned-example created
+	configmap/metallb-installation-preprovisioned-example created
+	```
+
+1. Use the wait command to monitor the cluster control-plane readiness:
+
+	```bash
+	kubectl wait --for=condition=ControlPlaneReady "clusters/${CLUSTER_NAME}" --timeout=30m
+	```
+	```sh
+	cluster.cluster.x-k8s.io/preprovisioned-example condition met
+	```
+
+1.  (Optional) If you have <a href="../create-secrets-and-overrides">overrides for your clusters</a>, you must specify the secret as part of the create cluster command.
+
+	<p class="message--note"><strong>NOTE: </strong> If these are not specified, the overrides for your nodes will not be applied. </p>
+
+	```bash
+	dkp create cluster preprovisioned --cluster-name ${CLUSTER_NAME} --control-plane-endpoint-host <control plane endpoint host> --control-plane-endpoint-port <control plane endpoint port, if different than 6443> --override-secret-name=$CLUSTER_NAME-user-overrides
+	```
+
+	<p class="message--note"><strong>NOTE: </strong>If your cluster is air-gapped or you have a local docker registry you must provide additional arguments when creating the cluster. </p>
+
+	```bash
+	export DOCKER_REGISTRY_ADDRESS="<https/http>://<registry-address>:<registry-port>"
+	export DOCKER_REGISTRY_CA="<path to the CA on the bastion>"
+	export DOCKER_REGISTRY_USERNAME="<username>"
+	export DOCKER_REGISTRY_USERNAME="<password>"
+	```
+		
+	- `DOCKER_REGISTRY_ADDRESS`: the address of an existing Docker registry accessible in the VPC that the new cluster nodes will be configured to use a mirror registry when pulling images.
+	- `DOCKER_REGISTRY_CA`: (optional) the path on the bastion machine to the Docker registry CA. Konvoy will configure the cluster nodes to trust this CA. This value is only needed if the registry is using a self-signed certificate and the AMIs are not already configured to trust this CA.
+	- `DOCKER_REGISTRY_USERNAME`: optional, set to a user that has pull access to this registry.
+	- `DOCKER_REGISTRY_PASSWORD`: optional if username is not set.
+
+		```bash
+		dkp create cluster preprovisioned --cluster-name ${CLUSTER_NAME} \
+		--control-plane-endpoint-host <control plane endpoint host> \
+		--control-plane-endpoint-port <control plane endpoint port, if different than 6443> \
+		--registry-mirror-url=${DOCKER_REGISTRY_ADDRESS} \
+		--registry-mirror-cacert=${DOCKER_REGISTRY_CA} \
+		--registry-mirror-username=${DOCKER_REGISTRY_USERNAME} \
+		--registry-mirror-password=${DOCKER_REGISTRY_PASSWORD}
+		```
+
+1. Depending on the cluster size, it will take a few minutes to be created. After the creation, use this command to get the Kubernetes kubeconfig for the new cluster and begin deploying workloads:
```

Comment:
```suggestion
1. Depending on the cluster size, it will take a few minutes to create. After the creation, use this command to get the Kubernetes kubeconfig for the new cluster and begin deploying workloads:
```